diff --git a/src/EventSubscriber/BuildIndexParams.php b/src/EventSubscriber/BuildIndexParams.php
index c8cbea4..606f9f2 100644
--- a/src/EventSubscriber/BuildIndexParams.php
+++ b/src/EventSubscriber/BuildIndexParams.php
@@ -162,6 +162,7 @@ class BuildIndexParams implements EventSubscriberInterface {
               'attachment' => [
                 'target_field' => '_ingest._value.attachment',
                 'field' => '_ingest._value.data',
+                'indexed_chars' => 500000,
               ],
             ],
           ],
